code/modules/SCP/SCPs/SCP-500.dm

```diff
@@ -4,21 +4,25 @@
 	classification = SAFE
 
 /obj/item/reagent_containers/pill/scp500
-	name = "SCP-500"
-	desc = "A pill that, when taken orally, cures the subject of all disease."
+	name = "pill"
+	desc = "A pill."
 	icon_state = "pill9"
 	SCP = /datum/scp/scp_500
 
 /obj/item/reagent_containers/pill/scp500/New()
 	. = ..()
 	reagents.add_reagent(/datum/reagent/adminordrazine, 1)
 
+/obj/item/reagent_containers/pill/scp500/afterattack(obj/target, mob/user, proximity) //can't dissolve it in water lmao!
+	return 0
+
+
 /obj/item/storage/pill_bottle/scp500
 	name = "pill bottle (Panacea)"
-	desc = "Contains pills of SCP-500."
+	desc = "This Pill Bottle has the words panacea written in it's cover, you also notice a ductaped note to it, it says - DUE TO UNAUTHORIZED USAGE OF 3 PILLS TO CURE STDs IN ZONE COMMANDERS THE REST OF THE PILLS HAVE BEEN DELIVERED TO REDACTED FOR PROPER CONTAINMENT."
```

Comment:
```suggestion
	desc = "This Pill Bottle has the words \'SCP-500\' written on its cover. There's a sticky note on it that reads, \'Due to repeated unauthorized usage, SCP-500 has been transferred to a secure Site. 8 pills have been kept for research purposes. Contact a RAISA official for more details.\'."
```
Much better description. The Foundation wouldn't redact something in this manner.